qml/Ping360Visualizer.qml

```diff
@@ -71,6 +71,51 @@ Item {
                     }
                 }
 
+                // Spinner that shows the head angle
+                Shape {
+                    id: shapeSpinner
+                    opacity: 1
+                    anchors.fill: parent
+                    vendorExtensionsEnabled: false
+                    property var angle: 0
+
+                    ShapePath {
+                        id: shapePathSpinner
+                        strokeWidth: -1
+                        property var centerX: waterfall.width/2
+                        property var centerY: waterfall.height/2
+
+                        fillGradient: LinearGradient {
+                            x1: x2*1.1
+                            y1: y2*1.5
+                            x2: waterfall.width/2
```

Comment:
`shapePathSpinner.centerX`?

Reply:
This is not related to the center point but with the radius of the ring. 
Added radius variable to be more clear.